app/cisc/sshconfig.go

```diff
@@ -0,0 +1,169 @@
+package main
+
+import (
+	"io/ioutil"
+	"os"
+	"strings"
+)
+
+/*
+A very basic ssh-config file reader and writer. It can handle comments in
+front of the 'Host' commands. When reading and writing, it is not guaranteed
+that the file stays the same.
+- sections always start with comment followed by "Host"
+- the configuration-part of a host has to start with a tab
+*/
+
+// SSHConfig holds all host-definitions
+type SSHConfig struct {
+	Host []*SSHHost
+}
+
+// NewSSHConfig takes a string which represents the ssh-config files, parses
```

Comment:
I am surely missing something here so don't hesitate to shut me down:
What are you calling ssh config file ? Is it the known_hosts ? In this case, this [library](https://godoc.org/golang.org/x/crypto/ssh#ParseKnownHosts) will do the trick just fine. I'm sure it's not that so can you give a link or something on which files you are reading ?